### PR TITLE
Fixed wrong overriden options warnings

### PR DESCRIPTION
**Pull request by miherlosev**
@kirovboris @AndreyBelym

### CODE REVIEW

**Comment by miherlosev** [comment]
@testcafe-build-bot retest

File: src/configuration/index.js [hunk 1]
```diff
@@ -59,10 +67,10 @@ export default class Configuration {
             console.log(warningMessage.errorConfigFileCannotBeParsed); // eslint-disable-line no-console
         }
 
-        await this._prepareOptions();
+        await this._afterLoadNormalization();
     }
 
-    async _prepareOptions () {
+    async _afterLoadNormalization () {
```
Comment (kirovboris) [inline src/configuration/index.js]: A function name should start from a verb,

**Comment by kirovboris** [comment]
@testcafe-build-bot /retest

**Comment by AndreyBelym** [comment]
I want to build the version from sources and test it, sorry for delays.

**Pull request merged**